tests/Fixer/Phpdoc/PhpdocTypesOrderFixerTest.php

```diff
@@ -522,6 +522,41 @@ public static function provideFixWithAlphaAlgorithmCases(): iterable
             '<?php /** @return A&B<X|Y|Z>&C&D */',
             '<?php /** @return A&D&B<X|Y|Z>&C */',
         ];
+
+        yield [
+            '<?php /** @param A|(B&C) */',
+            '<?php /** @param (C&B)|A */',
+        ];
+
+        yield [
+            '<?php /** @param A|(B&C) */',
+            '<?php /** @param (C&B)|A */',
+        ];
```

Comment:
This is exactly the same test case duplicated twice.

Reply:
Sorry / thx for the fix